ui/apps/platform/src/Components/EmailNotifier/EmailNotifierModal.tsx

```diff
@@ -36,16 +39,16 @@ function EmailNotifierModal({
             onCancel: onToggleEmailNotifierModal,
         });
 
-    function onSave(emailNotifier: EmailIntegrationFormValues): Promise<FormResponseMessage> {
+    function onSave(
+        emailNotifier: EmailIntegrationFormValues
+    ): Promise<FormResponseMessageWithData> {
         return createIntegration('notifiers', emailNotifier).then((response) => {
-            // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-            // @ts-ignore
-            return { isError: false, message: '', data: response?.data ?? null };
+            return { isError: false, message: '', data: response };
```

Comment:
The value of `data` in the response is never used, which is likely why the incorrect value here did not cause problems when ignored.